cms/static/sass/xmodule/_headings.scss

```diff
@@ -32,7 +32,7 @@ $headings-base-color:                       $gray-d2;
 
 
 %hd-2 {
-    font-size: em(18);
+    font-size: 1.1125em;
```

Comment:
How did you determine this value? I'm not familiar with the best practices for how font sizes should be specified.